app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,79 @@
+class FoodsController < ApplicationController
+  before_action :set_food, only: %i[show edit update destroy]
+
+  # GET /foods or /foods.json
+  def index
+    @foods = current_user.foods
+  end
+
+  # GET /foods/1 or /foods/1.json
+  def show; end
+
+  # GET /foods/new
+  def new
+    @food = Food.new
+  end
+
+  # GET /foods/1/edit
+  def edit; end
+
+  # POST /foods or /foods.json
+  def create
+    @food = Food.new(food_params)
+
+    # Set the user_id from the current_user
+    @food.user_id = current_user.id if user_signed_in?
+
+    respond_to do |format|
+      if @food.save
+        format.html { redirect_to foods_url, notice: 'Food was successfully created.' }
+        format.json { render :show, status: :created, location: @food }
+      else
+        format.html { render :new, status: :unprocessable_entity }
+        format.json { render json: @food.errors, status: :unprocessable_entity }
+      end
+    end
+  end
```

Comment:
- [ ] You have been doing a good job but I think you probably fixed an issue that led to another here because the previous reviewer did not mention that he was unable to add to the recipe, find the logged errors in the image below 
![image](https://github.com/Surafels/recipe-app/assets/86200356/29ba1f0e-219c-45a5-9a56-4d52498fdd28)

Kindly fix this issue for other suggestions to be checked in the next review.